src/render_helpers/shaders/color_interpol.frag

```diff
@@ -0,0 +1,42 @@
+
+vec4 color_linear(vec4 color) {
+  return vec4(
+    pow(color.r, 2.0),
+    pow(color.g, 2.0),
+    pow(color.b, 2.0),
+    color.a
+  );
```

Comment:
I wonder if it's more efficient to do something like `vec4(color.rgb * color.rgb, color.a)`